### ISSUE TITLE
Pan camera using hjkl

### ISSUE THREAD

**Issue by adam-r-kowalski** [adam-r-kowalski/studio#66]
Currently the only way to pan the camera on a laptop is using the mouse. Hot keys should be added so you can pan and zoom the camera using only your keyboard. This will help the keyboard only workflow and bring in vim users.
